From e8419e6bc7e851938d99ad356d987476b37c1129 Mon Sep 17 00:00:00 2001
From: Roman Dobrodii <rdobrodii@antmicro.com>
Date: Tue, 18 Apr 2023 18:25:56 -0700
Subject: [PATCH] Patch hardware/nvidia/platform/t23x/p3768/kernel-dts/cvb/tegra234-antmicro-job-camera-imx412c.dtsi

---
 1 file changed

--- /dev/null
+++ b/hardware/nvidia/platform/t23x/p3768/kernel-dts/cvb/tegra234-antmicro-job-camera-imx412c.dtsi
@@ -0,0 +1,3 @@
+// The contents of this file have been moved to the vc_mipi_nvidia
+// repository, and are now located in that repository at:
+//    src/devicetree/Antmicro_OrinNX/tegra234-camera-vc-mipi-cam.dtsi
